src/users/containers/user-details/user-details.component.ts

```diff
@@ -46,7 +54,8 @@ export class UserDetailsComponent implements OnInit, OnDestroy {
       this.handleDependanciesSubscription(users, route);
     });
 
-    this.userSubscription = this.user$.subscribe((user) => this.handleUserSubscription(user));
+
+    this.userSubscription = this.user$.subscribe((user) => this.handleUserSubscription(user, isfeatureEnabled$));
```

Comment:
No need for ((user) => ........ it can just be (user =>

Reply:
It was my code @ficklephil lol. Noted. 👍